tests/src/test/scala/org/apache/openwhisk/core/containerpool/yarn/test/YARNContainerFactoryTests.scala

```diff
@@ -35,29 +36,18 @@ import scala.concurrent.Await
 import scala.concurrent.duration._
 
 @RunWith(classOf[JUnitRunner])
-class YARNContainerFactoryTests extends Suite with BeforeAndAfter with FlatSpecLike {
-
+class YARNContainerFactoryTests extends Suite with BeforeAndAfter with FlatSpecLike with ExecHelpers {
   val images = Array(
-    ImageName("nodejs6action", Option("openwhisk"), Option("latest")),
-    ImageName("python3action", Option("openwhisk"), Option("latest")))
-
-  val runtimes: String = "{\"runtimes\":{" +
-    "\"nodejs\":[{\"kind\":\"nodejs:6\",\"image\":{\"prefix\":\"openwhisk\",\"name\":\"nodejs6action\",\"tag\":\"latest\"}}]," +
-    "\"python\":[{\"kind\":\"python:3\",\"image\":{\"prefix\":\"openwhisk\",\"name\":\"python3action\",\"tag\":\"latest\"}}]" +
-    "}}"
-
-  implicit val logging: PrintStreamLogging = new PrintStreamLogging()
-  implicit val whiskConfig: WhiskConfig = new WhiskConfig(
-    Map(wskApiHostname -> "apihost", runtimesManifest -> runtimes) ++ wskApiHost)
-
+    ImageName("nodejs6action", Option("openwhisk"), imageTag("nodejs:6")),
```

Comment:
The intent here was to provide a custom manifest to load so that it is decoupled from the actual runtime manifest.

I _think_ that the impression was that if you create a new whisk config object from the given manifest, the config is loaded when in fact it isn't (I should have pointed this out).

I prefer to restore the intended behavior and decouple the test manifest from the deployment manifest. I did this recently in another PR as an example, see here: https://github.com/apache/incubator-openwhisk/pull/4284/files#diff-497ffa2cac9fe692a1df3b53e9793a41R1645

Reply:
Trying out your suggestion, @rabbah.

Reply:
@rabbah, sorry didn't realize a custom manifest was intended here.

Reply:
As manifest is global the test needs to reset it back to default. It may be worthwhile to implement a [fixture][1] to take care of properly setting the manifest for specific test

[1]: http://www.scalatest.org/user_guide/sharing_fixtures#loanFixtureMethods

Reply:
You’re absolutely right Chetan. It’s my fault again for missing that.

Reply:
@chetanmeh, tried to implement a loan fixture. Seems like that is not the proper pattern to use as `ExecManifest` is a global Object.